src/models/OAuthModel.php

```diff
@@ -425,28 +423,59 @@ public function createUserFromTwitterUsername($clientId, $values)
      *
      * @param  string $clientId         aka consumer_key (of the joindin client)
      * @param  string $email            User's email address (that we just got back from authenticating them)
+     * @param  string $fullName         User's full name from Facebook
+     * @param  string $userName         Username to be created if not found
      * @return array|false              Array of access token and user uri on success or false or failure
      */
-    public function createAccessTokenFromTrustedEmail($clientId, $email)
+    public function createAccessTokenFromTrustedEmail($clientId, $email, $fullName = '', $userName = '')
     {
-        $sql = "select ID from user "
-            . "where email = :email "
-            . "and verified = 1";
+        $sql = "
```

Comment:
Do we really need to split that over multiple lines? Otherwise we probably should also split the function-definition over multiple lines…

Reply:
It's much more readable as it currently is. 

I can add a test none of this class / area is currently tested and would need parts re writing to be testable though